authlib/oauth2/rfc6749/authorization_server.py

```diff
@@ -329,6 +332,21 @@ def create_authorization_response(self, request=None, grant_user=None, grant=Non
         grant.execute_hook("after_authorization_response", response)
         return response
 
+    def create_pushed_authorization_response(self, request=None, grant_user=None, grant=None):
```

Comment:
I guess for this you could use the [register_endpoint](https://github.com/authlib/authlib/blob/bafecc468162babde7428d9829fc5b019c835a33/authlib/oauth2/rfc6749/authorization_server.py#L217) mechanism. You can take inspiration of [oidc UserInfo endpoint](https://github.com/authlib/authlib/blob/bafecc468162babde7428d9829fc5b019c835a33/authlib/oidc/core/userinfo.py) for example.